.github/workflows/ci.yml

```diff
@@ -56,23 +56,18 @@ jobs:
       - uses: actions/checkout@v1
         with:
           submodules: true
-      # - name: install LLVM on Linux
-      #   if: matrix.os == 'ubuntu-latest'
-      #   run: sudo apt-get install llvm-dev libclang-dev clang
-      # - name: install LLVM on Mac
-      #   if: matrix.os == 'macOS-latest'
-      #   run: brew install llvm
-      - name: install LLVM on Windows
+      - name: Run cargo test (Windows)
         if: matrix.os == 'windows-latest'
-        run: choco install llvm -y
-      - uses: actions-rs/toolchain@v1
-        with:
-          toolchain: stable
-          profile: minimal
-          override: true
-      - uses: actions-rs/cargo@v1
-        with:
-          command: test
+        shell: powershell
+        run: |
+          iex (new-object net.webclient).downloadstring('https://get.scoop.sh')
+          scoop install llvm --global
```

Comment:
Is there any specific reason to switch to scoop here?

Zebra's CI still uses chocolates, and in general we'd like to limit the number of CI dependencies.

Reply:
One of the advantages of `scoop` that LLVM installed within 30s compare to almost 200s with `chocolatey` :(
I'll switch back to `chocolatey` with env fixes.

Reply:
Fixed.

Reply:
> One of the advantages of `scoop` that LLVM installed within 30s compare to almost 200s with `chocolatey` :(

Wow that is a big difference. It would be very useful for our engineers to have 3 minutes faster CI builds.

As long as scoop has decent uptime, I'd be happy to take separate PRs to switch to scoop for:
- zebra
- zcash_script
- ed25519-zebra

and anything else we're actively developing. (@dconnolly will know if there are any extra crypto crates that we should switch over now.)

I'd like a separate PR so we don't block crate dependency changes on CI environment changes.